R/lime.R

```diff
@@ -160,16 +160,14 @@ select_tree <- function(x, y, weights, n_features) {
 select_f_lp <- function(x, y, weights, n_features) {
   shuffle_order <- sample(length(y)) # glm is sensitive to the order of the examples
   fit <- glmnet(x[shuffle_order,], y[shuffle_order], weights = weights[shuffle_order], alpha = 1, nlambda = 300)
-  # In case that no model with correct n_feature size was found
-  if (all(fit$df != n_features)) {
-    stop(sprintf("No model with %i features found with lasso_path. Try a different method.", n_features))
-  }
   has_value <- apply(coef(fit)[-1, ], 2, function(x) x != 0)
   f_count <- apply(has_value, 2, sum)
   row <- which(f_count >= n_features)[1]
   features <- which(has_value[, row])
```

Comment:
hmm.. wouldn't you get the same result if you simply did `which(rev(f_count) <= n_features)[1]`?

Honest question, but can you try it out?

Reply:
referencing line 165: `row <- which(f_count >= n_features)[1]`

Reply:
Hmmm... didn't work. I think I would also have to reverse the has_value vector too...

Reply:
Ah, yeah - of course... let's just keep it as is for now